bigquery/google/cloud/bigquery/client.py

```diff
@@ -379,8 +380,8 @@ def create_table(self, table, exists_ok=False, retry=DEFAULT_RETRY):
             google.cloud.bigquery.table.Table:
                 A new ``Table`` returned from the service.
         """
-        if isinstance(table, str):
-            table = TableReference.from_string(table, default_project=self.project)
+        table = _str_or_ref_to_table_ref(table, default_project=self.project)
```

Comment:
Renamed to `_table_arg_to_table_ref` in https://github.com/googleapis/google-cloud-python/pull/7560/commits/3ab95d4ad7da8b2be27171716e294f024882536d.

It converts more types such as `Table` to `TableReference`, but I still found it really convenient for the `copy_table` method to allow lists to pass through unchanged. The reason for that is it's actually pretty difficult to disambiguate a string from a list of stuff since they're both iterable sequences.